Bindings/Java/Matlab/Utilities/plotMuscleFLCurves.m

```diff
@@ -1,40 +1,16 @@
-% -------------------------------------------------------------------------- %
-%                            plotMuscleFLCurves.m                            %
-% -------------------------------------------------------------------------- %
-% The OpenSim API is a toolkit for musculoskeletal modeling and simulation.  %
-% See http://opensim.stanford.edu and the NOTICE file for more information.  %
-% OpenSim is developed at Stanford University and supported by the US        %
-% National Institutes of Health (U54 GM072970, R24 HD065690) and by DARPA    %
-% through the Warrior Web program.                                           %
-%                                                                            %
-% Copyright (c) 2005-2017 Stanford University and the Authors                %
-% Author(s): James Dunne                                                     %
-%                                                                            %
-% Licensed under the Apache License, Version 2.0 (the "License"); you may    %
-% not use this file except in compliance with the License. You may obtain a  %
-% copy of the License at http://www.apache.org/licenses/LICENSE-2.0.         %
-%                                                                            %
-% Unless required by applicable law or agreed to in writing, software        %
-% distributed under the License is distributed on an "AS IS" BASIS,          %
-% WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.   %
-% See the License for the specific language governing permissions and        %
-% limitations under the License.                                             %
-% -------------------------------------------------------------------------- %
-
+function plotMuscleFLCurves(modelpath)
+% Plot the force length curve of a muscle in the Model
```

Comment:
If the model path is to a Model with multiple muscles, how is a particular muscle specified? Perhaps state that the user is prompted for the muscle's name.